backend/services/rabbit-service/src/consumer.ts

```
@@ -0,0 +1,17 @@
+
+export async function consumeMessageFromQueue(channel: Channel, queueName: string, callback: (msg: ConsumeMessage | null) => void): Promise<void> {
+    try {
+        await channel.assertQueue(queueName, { durable: true });
+        await channel.consume(queueName, (msg) => {
+            if (msg !== null) {
+                callback(msg);
+                channel.ack(msg); // Acknowledge the message after processing
+            }
+        });
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)